sf_examples/envpool_examples/mujoco_examples/mujoco/envpool_mujoco_utils.py

```diff
@@ -0,0 +1,55 @@
+try:
+    import envpool
+except ImportError as e:
+    print(e)
+    print("Trying to import envpool when it is not install. install with 'pip install envpool'")
+
+import gym
+
+from sample_factory.utils.utils import is_module_available
+from sf_examples.envpool_examples.envpool_wrappers import EnvPoolTo5Tuple
+
+
+def mujoco_available():
+    return is_module_available("mujoco")
+
+
+class MujocoSpec:
+    def __init__(self, name, env_id):
+        self.name = name
+        self.env_id = env_id
+
+
+MUJOCO_ENVS = [
+    MujocoSpec("mujoco_hopper", "Hopper-v4"),
```

Comment:
I think this is identical to regular mujoco example too - so maybe just import from there? :)